templates/swagger/v1_json.tmpl

```diff
@@ -16584,6 +16584,35 @@
       },
       "x-go-package": "code.gitea.io/gitea/modules/structs"
     },
+    "IssueFormField": {
```

Comment:
Does any API use this struct? If it's only an internal struct, it shouldn't be exposed to API or change the API swagger document

Reply:
Yes, a new field `body` typed `[]IssueFormField` has been added to `IssueTemplate`, users can get them via API `/repos/{owner}/{repo}/issue_templates`.